io.github.junalmeida.webapps_manager.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "io.github.junalmeida.webapps_manager",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
```

Comment:
6.8 is the latest runtime.

Reply:
I know, but it does not work and I have this issue open against the BaseApp project. https://github.com/flathub/io.qt.PySide.BaseApp/issues/11
I plan to upgrade once I can test it runs successfully.